src/opening-period/page/CreateNewOpeningPeriodPage.scss

```diff
@@ -16,45 +16,26 @@
   label {
     display: block;
     margin-top: $spacing-m;
+    font-family: HelsinkiGrotesk-Medium, Arial, sans-serif;
   }
 
   input {
     display: block;
   }
 
   h3 {
+    margin-top: 0;
     margin-bottom: 0;
   }
 
   margin-top: $spacing-xl;
 }
 
-.opening-period-optional-description {
-  padding: $spacing-layout-2-xs;
-  width: calc(0.5808333 * 100%);
-  height: 164px;
-  border: 2px solid rgb(128, 128, 128);
-  margin-bottom: $spacing-layout-2-xs;
-
-  @media screen and (max-width: $breakpoint-m) {
-    width: 100%;
-  }
-}
-
 .opening-period-section-title {
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
```

Comment:
Is this needed?  There seems to be the same CSS rule in index.scss for h3-element

Reply:
- [ ] Check if this font-family style definition is needed, and remove it if it is not

Reply:
It seemed to be redundant, so I removed it!